XiEditor/CoreConnection.swift

```diff
@@ -18,14 +18,14 @@ class CoreConnection {
 
     var inHandle: FileHandle  // stdin of core process
     var recvBuf: Data
-    var callback: (AnyObject) -> ()
+    var callback: (Any) -> Any?
```

Comment:
Changed`AnyObject` to `Any` for consistency reasons (also, `init` method below expects `Any` instead of `AnyObject`)